examples/vision/nerf.py

```diff
@@ -0,0 +1,778 @@
+"""
+Title: 3-D Volumetric Rendering using NeRF
+Authors: [Aritra Roy Gosthipaty](https://twitter.com/arig23498), [Ritwik Raha](https://twitter.com/ritwik_raha)
+Date created: 2021/08/09
+Last modified: 2021/08/09
+Description: Minimal implementation of volumetric rendering as shown in NeRF.
+"""
+"""
+## Introduction
+
+In this example, we minimally implement the research paper
+[**NeRF: Representing Scenes as Neural Radiance Fields for View Synthesis**](https://arxiv.org/abs/2003.08934)
+by Ben Mildenhall et. al. The authors have proposed an ingenious way
+to **synthesise novel views** of a scene by modelling the *volumetric
+scene function* through a neural network.
+
+To construct an intuition, let me ask you a simple question. *What
+will happen if we input the coordinates of an image to a neural
+network and ask it to predict the color at that coordinate?*
```

Comment:
It isn't clear what the "coordinates of an image" means, please edit the text to make it clearer.

Reply:
I have replaces the term `coordinates` with `pixel position`. Does that seem to make it clearer?